src/auto-evo/simulation/SimulationCache.cs

```diff
@@ -23,6 +29,12 @@ public class SimulationCache
         private readonly Dictionary<(TweakedProcess, BiomeConditions), ProcessSpeedInformation> cachedProcessSpeeds =
             new();
 
+        private readonly Dictionary<MicrobeSpecies, float[]> cachedPredationToolsRawScores = new();
+        private readonly Dictionary<(OrganelleTemplate, BiomeConditions, Compound), float>
```

Comment:
I think this cache size will blow up pretty easily as each species has one OrganelleTemplate per organelle, and those instances are not equal when new attempted organelles are used. Would caching based on OrganelleDefinition work? Those wouldn't pollute the cache with single use data.

Reply:
I think it would. I'll need to have a closer look at it to be sure though.